catalog/core/catalog-core-solrcommands/src/main/java/org/codice/ddf/commands/solr/SolrCommands.java

```diff
@@ -33,11 +38,17 @@
 
     protected PrintStream console = System.out;
 
+    protected ConfigurationAdmin configurationAdmin;
+
+    protected static final String ZOOKEEPER_HOSTS_PROP = "solr.cloud.zookeeper";
```

Comment:
ZOOKEEPER_HOSTS_PROP can but SYSTEM_PROPERTIES_PATH is used in BackupCommand.java